xml/tuning-ceph-tuning.xml

```diff
@@ -676,21 +704,21 @@ radosgw-admin bucket reshard --bucket={bucket name} --num-shards={prime number}
    <sect2>
      <title>Parallel I/O Requests</title>
      <para>
-       By default, the &rgw; process is limited to eight simultaneous I/O
+       By default, the &ogw; process is limited to eight simultaneous I/O
        operations for the index. This can be adjusted with the
        <literal>rgw_bucket_index_max_aio</literal> parameter.
      </para>
      </sect2>
      <sect2>
        <title>Window Size</title>
      <para>
-       When working with larger objects, increasing the size of the &rgw;
+       When working with larger objects, increasing the size of the &ogw;
        windows for <literal>put</literal> and get can help with performance.
```

Comment:
```suggestion
       windows for <literal>put</literal> and <literal>get</literal> can help with performance.
```

Reply:
Also note 2 typos in a later line to which I cannot comment directly:

> allowing up to two devices or nods to fail while still proecting

allowing up to two devices or **nodes** to fail while still **protecting**